Rules.md

```diff
@@ -19543,6 +19544,46 @@ class BBBB {}
 
 
 
+## Vertical Whitespace Between Cases
+
+Identifier | Enabled by default | Supports autocorrection | Kind | Minimum Swift Compiler Version
+--- | --- | --- | --- | ---
+`vertical_whitespace_between_cases` | Disabled | Yes | style | 3.0.0 
+
+Include a vertical whitespace (empty line) between cases in switch statements.
+
+### Examples
+
+<details>
+<summary>Non Triggering Examples</summary>
+
+```swift
+    switch x {
+    case 0..<5:
+        print("x is valid")
+
+    default:
+        print("x is invalid")
+    }
+```
+
+</details>
+<details>
+<summary>Triggering Examples</summary>
+
+```swift
+    switch x {
+    case 0..<5:
```

Comment:
This triggering examples are not showing where the violation is triggered. Maybe this should be added, to match the other rules.

Reply:
Done.